configs/components/marionette-collective.rb

```diff
@@ -73,6 +49,29 @@
     fail "need to know where to put service files"
   end
 
+  if (platform.servicetype == "sysv" && platform.is_rpm?) || platform.is_aix?
```

Comment:
If I'm following the flow correctly, I think that matches what we're doing now? https://github.com/puppetlabs/puppet-agent/pull/738/files#diff-9ae79323449fab62c617785704463a77L29.